CMakeLists.txt

```diff
@@ -148,7 +148,6 @@ endif()
 find_package(Threads REQUIRED)
 
 list(APPEND EXTERNAL_LIBS glog)
-list(APPEND EXTERNAL_LIBS snappy)
```

Comment:
Hmmmm. Why is it deleted?

Reply:
```
ld: warning: ignoring duplicate libraries: '_deps/snappy-build/libsnappy.a', 'clang_17.0_cxx17_64_relwithdebinfo/libtbb.a'
```
I found a warning when compiling with clang 17. It appears that deleting this line does not affect the compilation process.

Reply:
I'd prefer to explicit list this dep instead of depend on transitive dep.

Reply:
fixed